### PR TITLE
Add statistics block and generate reports in frontend

### PR DESCRIPTION
**Pull request by DarkFlamingo**

### CODE REVIEW

**Comment by DarkFlamingo** [comment]
Block style
image
File style
image

File: frontend/src/scenes/ProjectPage/components/Statistics/index.tsx [hunk 1]
```diff
@@ -1,76 +1,106 @@
...
-        dataSets={params.dataSets}
-        options={params.options}
-        height="230px"
-      />
-    </div>
+    <Col className={classes['statistics-wrapper']}>
+      <div className={classes['statistics-card-wrapper']}>
+        <Row>
+          <header className={classes['statistics-topic']}>
+            Global Statistics
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/ProjectPage/components/Statistics/index.tsx]: Localization

File: frontend/src/scenes/ProjectPage/reducer.ts [hunk 2]
```diff
@@ -1,23 +1,34 @@
...
+  fetchStatistics,
+  setReaction,
+  setWatch
+} from './actions';
 
 export interface ProjectPageState {
   isLoading: boolean;
   project: ProjectPage;
   isInputLoading: boolean;
+  statistics: Statistics;
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/ProjectPage/reducer.ts:24]: Maybe it's better to use undefined instead?

File: frontend/src/services/project.ts [hunk 3]
```diff
@@ -17,6 +33,18 @@ export const getProject = (
...
 ): Promise<Project> => api.get({ url: `projects/${params.id}`, params });
 
-export const setReaction = (params : SetReactionProps) => api.post({ url: 'projects/reaction', params });
+export const setReaction = (params: SetReactionProps) => api.post({ url: 'projects/reaction', params });
 
 export const setWatch = (params: SetWatchProps) => api.post({ url: 'projects/watch', params });
+
+export const getProjectsStatistics = async ({
+  id,
+  timeInterval
```
Comment (yevhenii-moroziuk) [inline frontend/src/services/project.ts]: no need to use deconstruction here

File: frontend/src/services/project.ts [hunk 4]
```diff
@@ -17,6 +33,18 @@ export const getProject = (
...
+}: FetchStatistics) => {
+  const statistics: Statistics = await api.get({
+    url: `projects/${id}/statistics`,
+    params: { id, timeInterval }
+  });
+  console.log(statistics);
```
Comment (yevhenii-moroziuk) [inline frontend/src/services/project.ts]: log

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/ProjectPage/reducer.ts (hunk 2 above)
Comment (DarkFlamingo) [reply frontend/src/scenes/ProjectPage/reducer.ts:24]: Instead of the default value of the statistics ?

Comment (DarkFlamingo) [reply frontend/src/scenes/ProjectPage/reducer.ts:24]: Using undefined there are some problems with chart js params

**Review by yevhenii-moroziuk — APPROVED** [review]

**Pull request merged**